ivadomed/losses.py

```diff
@@ -1,16 +1,54 @@
 import torch
 import torch.nn as nn
-import torch.nn.functional as F
+
+
+class MultiClassDiceLoss(nn.Module):
+    # Inspired from https://arxiv.org/pdf/1802.10508.pdf
+    def forward(self, prediction, target):
+        smooth = 1.0
+        dice_per_class = 0
+        n_classes = prediction.shape[1]
+        for i in range(n_classes):
+            iflat = prediction[:, i, ].reshape(-1)
+            tflat = target[:, i, ].reshape(-1)
+            intersection = (iflat * tflat).sum()
+            dice_per_class += intersection / (iflat.sum() + tflat.sum() + smooth)
+        return -(2.0 * dice_per_class) / n_classes
+
+
+class CombinedDiceLoss(nn.Module):
+    """
+    :param classes_of_interest: list containing the index of a class which dice will be added to the loss.
+    :param multiclass_dice: boolean representing whether or not multiclass dice is included in the current loss.
+    :param dice: boolean representing whether or not dice is included in the current loss.
```

Comment:
My understanding is that these three params are mutually exclusive, is it correct? eg [XX], False, False OR [], True, False OR [], False, True.
Could we send only one param instead?eg  [0] for monoclass, [0, 2] for class 1 and 2, [0, 1, 2] for all classes.

Reply:
You can have all three at the same time: [xx], True, True -> yes, one param would be best I'll change that!

Reply:
I changed it for a list of the params, I think it might affect clarity? Tell me what you think!